README.md

```diff
@@ -0,0 +1,86 @@
+# RateLimitBundle
+This bundle provides an easy way to protect your project by limiting access to your controllers.
+
+## Install the bundle
+```bash
+composer require bedrock/rate-limit-bundle
+```
+
+Update your _config/bundles.php_ file to add the bundle for all env
+```php
+<?php
+
+return [
+    ...
+    Bedrock\Bundle\RateLimitBundle\RateLimitBundle::class => ['all' => true],
+    ...
+];
+```
+
+### Configure the bundle
+Add the _config/packages/bedrock_rate_limit.yaml_ file with the following data.
+```yaml
+bedrock_rate_limit:
+    limit_by_route: true|false # false by default
+    limit: 25 # 1000 requests by default
+    period: 600 # 60 seconds by default
+    display_headers: true|false # false by default
+```
+By default, the limitation is common to all routes annotated `@RateLimit()`. 
+For example, if you keep the default configuration and you configure the `@RateLimit()` annotation in 2 routes. Users will be allowed to call 999 times the first route and only 1 time the second route.
```

Comment:
```suggestion
For example, if you keep the default configuration and you configure the `@RateLimit()` annotation in 2 routes. Web Users could consume all authorized call on the first route, and the second route will be not authorized
```
/!\ franglais

Reply:
> will be not authorized

will not be authorized

Reply:
> call 
calls

Even if I think it was better explained on the original version